### PR TITLE
Refactoring/warnings count

### PR DESCRIPTION
**Pull request by bnsports**

### CODE REVIEW

File: fearless/Modules/Staking/SelectValidatorsFlow/SelectValidatorsConfirm/Flow/RelaychainInitiated/SelectValidatorsConfirmRelaychainInitiatedViewModelFactory.swift [hunk 1]
```diff
@@ -88,17 +88,13 @@ extension SelectValidatorsConfirmRelaychainInitiatedViewModelFactory: SelectVali
 
     func buildViewModel(
         viewModelState: SelectValidatorsConfirmViewModelState,
-        asset: AssetModel
+        asset _: AssetModel
```
Comment (Viscount88) [inline fearless/Modules/Staking/SelectValidatorsFlow/SelectValidatorsConfirm/Flow/RelaychainInitiated/SelectValidatorsConfirmRelaychainInitiatedViewModelFactory.swift:91]: remove parameter

File: fearless/Modules/Staking/SelectValidatorsFlow/SelectValidatorsConfirm/SelectValidatorsConfirmViewController.swift [hunk 2]
```diff
@@ -118,7 +118,7 @@ final class SelectValidatorsConfirmViewController: UIViewController, ViewHolder,
             rootView.rewardDestinationView.valueTop.text = R.string.localizable
                 .stakingRestakeTitle(preferredLanguages: selectedLocale.rLanguages)
             rootView.removePayoutAccountIfNeeded()
-        case let .payout(icon, title, address):
+        case let .payout(_, title, address):
```
Comment (Viscount88) [inline fearless/Modules/Staking/SelectValidatorsFlow/SelectValidatorsConfirm/SelectValidatorsConfirmViewController.swift:121]: remove from case?

File: fearless/Modules/Wallet/Commands/WalletAccountOpenCommand.swift [hunk 3]
```diff
@@ -1,54 +1,54 @@
...
-
-    func execute() throws {
-//        let controller = UIAlertController.presentAccountOptions(
-//            address,
-//            chain: chain,
-//            locale: locale,
-//            copyClosure: copyAddress,
-//            urlClosure: present(url:)
-//        )
+// import Foundation
```
Comment (Viscount88) [inline fearless/Modules/Wallet/Commands/WalletAccountOpenCommand.swift:1]: remove

File: fearless/Modules/Wallet/Commands/WalletEventOpenCommand.swift [hunk 4]
```diff
@@ -1,77 +1,77 @@
...
-    }
-
-    private func present(url: URL) {
-        let webController = WebViewFactory.createWebViewController(for: url, style: .automatic)
-        let command = commandFactory?.preparePresentationCommand(for: webController)
-        command?.presentationStyle = .modal(inNavigation: false)
-        try? command?.execute()
-    }
-}
+// import Foundation
```
Comment (Viscount88) [inline fearless/Modules/Wallet/Commands/WalletEventOpenCommand.swift:1]: remove or fix

File: fearless/Modules/Wallet/Receive/View/ReceiveViewFactory.swift [hunk 5]
```diff
@@ -1,61 +1,61 @@
...
-        }
-
-        let infoTitle = R.string.localizable
-            .walletReceiveDescription(preferredLanguages: locale.rLanguages)
-        receiveView?.infoLabel.text = infoTitle
-
-        return receiveView
-    }
-}
+// import Foundation
```
Comment (Viscount88) [inline fearless/Modules/Wallet/Receive/View/ReceiveViewFactory.swift:1]: remove

File: fearlessIntegrationTests/ExtrinsicServiceTests.swift [hunk 6]
```diff
@@ -1,186 +1,186 @@
...
-            case let .failure(error):
-                XCTFail(error.localizedDescription)
-                feeExpectation.fulfill()
-            }
-        }
-
-        wait(for: [feeExpectation], timeout: 20)
-    }
-}
+//import XCTest
```
Comment (Viscount88) [inline fearlessIntegrationTests/ExtrinsicServiceTests.swift:1]: remove or fix

File: fearlessTests/Common/DataProvider/AnyProviderAutoClearTests.swift [hunk 7]
```diff
@@ -1,30 +1,30 @@
...
-
-        XCTAssertNotNil(provider)
-
-        cleaner.clear(dataProvider: &provider)
-
-        XCTAssertNil(provider)
-    }
-
-}
+//import XCTest
```
Comment (Viscount88) [inline fearlessTests/Common/DataProvider/AnyProviderAutoClearTests.swift:1]: remove or fix

File: fearlessTests/Common/Migration/NetworkConnectionsMigratorTests.swift [hunk 8]
```diff
@@ -1,26 +1,26 @@
...
-        } catch {
-            XCTFail(error.localizedDescription)
-        }
-
-        // then
-        let connectionTypeAfterMigration = settings.selectedConnection.type
-        XCTAssert(connectionTypeAfterMigration == connectionTypeBeforeMigration)
-    }
-}
+//import XCTest
```
Comment (Viscount88) [inline fearlessTests/Common/Migration/NetworkConnectionsMigratorTests.swift:1]: remove or fix

**Review by DRadmir — CHANGES_REQUESTED** [review]

File: fearless/Modules/Staking/SelectValidatorsFlow/SelectValidatorsConfirm/Flow/PoolInitiated/SelectValidatorsConfirmPoolInitiatedViewModelFactory.swift [hunk 9]
```diff
@@ -102,29 +102,8 @@ extension SelectValidatorsConfirmPoolInitiatedViewModelFactory: SelectValidators
...
-                title: account.username,
-                address: account.address
-            )
-        }
-
-        return LocalizableResource { locale in
-            let amount = amountFormatter.value(for: locale).string(from: state.amount as NSNumber)
-
-            return SelectValidatorsConfirmViewModel(
+        return LocalizableResource { _ in
```
[the remark this answers is not in the document]
Comment (bnsports) [reply fearless/Modules/Staking/SelectValidatorsFlow/SelectValidatorsConfirm/Flow/PoolInitiated/SelectValidatorsConfirmPoolInitiatedViewModelFactory.swift:105]: one of implementations use localizable resource

File: fearless/Modules/Staking/SelectValidatorsFlow/SelectValidatorsConfirm/SelectValidatorsConfirmViewController.swift (hunk 2 above)
Comment (bnsports) [reply fearless/Modules/Staking/SelectValidatorsFlow/SelectValidatorsConfirm/SelectValidatorsConfirmViewController.swift:121]: it can be used in other places

File: Jenkinsfile [hunk 10]
```diff
@@ -7,8 +7,8 @@ def jobParams = [
 
 def appPipline = new org.ios.AppPipeline(
     steps: this, 
-    appTests: false,
+    appTests: true,
     appPushNoti: true,
     jobParams: jobParams,
-    label: 'macos-ios-1-2')
+    label: 'macOS-latest')
```
Comment (C4tWithShell) [inline Jenkinsfile:13]: Why did you change it? We does not have runner called macOS-latest. Build will be stuck because of it

**Review by C4tWithShell — CHANGES_REQUESTED** [review]

**Review by DRadmir — APPROVED** [review]

File: Jenkinsfile (hunk 10 above)
Comment (bnsports) [reply Jenkinsfile]: Bao said me change it to macos-latest. and it works previously. Anyway I've changed back

**Review by C4tWithShell — APPROVED** [review]
